### PR TITLE
General role refactoring

### PR DESCRIPTION
**Pull request by atosatto**
- General Role refactoring.
- Deprecating URL-based installation in favor of repositories management.
- README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+metronome_configure_repo: False
+metronome_apt_repo: ""     # Metronome APT repository URL
+metronome_gpg_key_url: ""  # Metronome GPG key URL
+metronome_gpg_key_id: ""   # Metronome GPG key ID
+metronome_yum_repo: ""     # Metronome YUM repository URL
+```
+
+If `metronome_configure_repo` is True, the role will add the metronome repository to the system.
+The `metronome_apt_repo` and `metronome_yum_repo` variables should contain the URL of the repository according to the target operating system.
+The specification of the GPG key through the `metronome_gpg_key_url` and `metronome_gpg_key_id` variables, altought recommended, is not mandatory.
```
Comment (pieterlexis) [inline README.md]: s/althought/although/

File: README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+metronome_webserver_address: "[::]:8000"
+```
+
+The listen address for the embedded web-server.
+
+```
+metronome_stats_directory: '/var/lib/metronome'
+```
+
+The directory in which metronome will store the collected stats file.
```
Comment (pieterlexis) [inline README.md]: files

File: README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+metronome_replace_local_js: True
+metronome_local_js_scheme: 'http'
+metronome_local_js_address: "{{ ansible_default_ipv4['address'] }}"
+metronome_local_js_url: '/'
+```
+
+If `metronome_replace_local_js` is set to True, the role will override the default metronome `local_js` in order to make
```
Comment (pieterlexis) [inline README.md]: `local.js`

**Review by pieterlexis — APPROVED** [review]
Only some doc-nits

**Comment by atosatto** [comment]
@pieterlexis thanks for your review! :)

**Pull request merged**